CHANGELOG

```diff
@@ -36,10 +36,13 @@ v3.17 (Month 2020)
     - Fix textile preview not showing on issues with very long text
     - Long items_table dropdown menus not scrollable
     - Long project names interfering with search bar expansion
+    - Long unbroken table cell text in textile elements overflows
```

Comment:
These are in the wrong version block now.

Reply:
moved